monkey/infection_monkey/puppet/mock_puppet.py

```diff
@@ -135,7 +135,7 @@ def fingerprint(
         return empty_fingerprint_data
 
     def exploit_host(
-        self, name: str, host: str, options: Dict, interrupt: threading.Event
+        self, name: str, host: object, options: Dict, interrupt: threading.Event
```

Comment:
Just a note to remember to solve the circular dependency and change `object` to `VictimHost`.